tests/integration/test_crunch_cube.py

```diff
@@ -1246,17 +1246,25 @@ def test_cat_x_cat_as_array_prune_cols(self):
             [0, 1, 0, 0],
         ])
         actual = cube.as_array(prune=False)
+        np.testing.assert_array_equal(actual, expected)
         expected = np.array([
             [2, 2, 1],
             [0, 1, 2],
             [0, 2, 0],
             [0, 2, 1],
             [0, 1, 0],
         ])
+        # Use API method parameter to prune
         table = cube.as_array(prune=True)
         actual = table[:, ~table.mask.all(axis=0)][~table.mask.all(axis=1), :]
         np.testing.assert_array_equal(actual, expected)
 
+        # Use construction arguments to prune
+        cube = CrunchCube(CAT_X_CAT_WITH_EMPTY_COLS, prune=True)
+        table = cube.as_array()
+        actual = table[:, ~table.mask.all(axis=0)][~table.mask.all(axis=1), :]
```

Comment:
you don't need to redefine this, you have exactly that variable on line 1259 sitting waiting for you

Reply:
same goes for all the other expectations there